src/index.ts

```diff
@@ -118,6 +125,28 @@ export default class NewRelicLambdaLayerPlugin {
     await Promise.all(promises);
   }
 
+  public async checkIntegrationRole() {
+    const { accountId } = this.config;
```

Comment:
It seems that in the ReadMe we're recommending people remove `accountId` in favor of `trustedAccountKey`. That should probably replace accountId here, too.

Reply:
hey @mrickard based on https://github.com/newrelic/serverless-newrelic-lambda-layers/blob/master/README.md#accountid-required it seems like account id is required but trusted account is only necessary when an account is a sub account. Can you point me to where you're seeing that trusted account is preferred?

Reply:
@katiewest820 Yes, you're correct. The trusted account key is only necessary for a subaccount and would be the subaccount's parent account id.